cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/cube/CubeQuery.java

```diff
@@ -16,13 +16,12 @@
 
 package co.cask.cdap.metrics.store.cube;
```

Comment:
So this class remains in cdap-watchdog? How do we use a ``CubeQuery`` from data-fabric? Are we not supposed to use CubeQuery?